packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -64,10 +64,7 @@ function Export ({ className, match: { params: { address } } }: Props): React.Re
       />
       <div className={className}>
         <Address address={address}>
-          <Warning
-            className='movedWarning'
-            isDanger
```

Comment:
This is a warning, not an error, removing the isDanger. The Button is still red.